core/controllers/question_editor.py

```diff
@@ -25,12 +25,47 @@
 import feconf
 
 
+class QuestionCreationHandler(base.BaseHandler):
+    """A handler that creates the question model given a question dict."""
+
+    @acl_decorators.can_manage_question_skill_status
+    def get(self):
+        """Handles GET requests."""
+        if not feconf.ENABLE_NEW_STRUCTURES:
+            raise self.PageNotFoundException
+
+        new_question_id = question_services.get_new_question_id()
+        self.render_json({
+            'questionId': new_question_id
```

Comment:
Generally I thought we used underscores for the keys sent using render_json? This also helps a little in distinguishing backend dicts from frontend domain objects in the frontend.

Reply:
Yeah, we do, missed this. Anyway, removed this function